src/app/services/explorer/explorer.service.ts

```diff
@@ -224,4 +240,16 @@ export class ExplorerService {
     return this.api.getTransaction(transactionId).pipe(
       map(response => parseGetTransaction(response)));
   }
+
+  /**
+   * Returns the name associated to an address, enclosed in parentheses. If no name has been
+   * associated to the address, returns an empty string.
+   */
+  getAddressName(address: string): string {
+    if (this.namedAddressesMap.has(address)) {
+      return ' (' + this.namedAddressesMap.get(address) + ')';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin-explorer/pullRequest?prid=7163626)
```suggestion
      return " (" + this.namedAddressesMap.get(address) + ')';
```